bin/main.rb

```diff
@@ -1,2 +1,66 @@
 #!/usr/bin/env ruby
-puts 'Hello World!'
+
+# Board
+
+board = [
+  ['-', '-', '-'],
+  ['-', '-', '-'],
+  ['-', '-', '-']
+]
+
+puts(board.map { |x| x.join(' | ') })
+
+count_free_space = 9
+
+while count_free_space >= 1
```

Comment:
- [ ] [REQUIRED] Usage of out-of-nowhere numbers to terminate the loop is not advised. You should make use of a terminating variable like in the example below:
```
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```